addons/account/models/account_invoice.py

```diff
@@ -416,7 +416,7 @@ def _get_vendor_display_info(self):
                     vendor_display_name = _('From: ') + invoice.source_email
                     invoice.invoice_icon = '@'
                 else:
-                    vendor_display_name = ('Created by: ') + invoice.sudo().create_uid.name
+                    vendor_display_name = _('Created by: %s') % invoice.sudo().create_uid.name
```

Comment:
I think it's better to leave `%s` out of translations if possible. And also the spaces.

Reply:
Not really, in some languages like Japanese, the order of words changes, this allows to make translations where the placeholder has a different position